### PR TITLE
Added github actions

### PR DESCRIPTION
**Pull request by lb1c**
Added Github internal CI called "Actions" to the codebase.

### CODE REVIEW

**Comment by oterral** [comment]
@lb1c  where do we see if the build pass or not ?

**Comment by lb1c** [comment]
@oterral currently only in my fork, until it is merged into the geops repo:
https://github.com/lb1c/trafimage-maps/actions 

in the future it'll be visible in every pull request and in addition to that here:
https://github.com/geops/trafimage-maps/actions

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: yarn install, build, and test
+      run: |
+        npm install -g yarn
+        yarn install
+        yarn lint
+        yarn build:es
```
Comment (oterral) [inline .github/workflows/nodejs.yml]:
```suggestion
        yarn build
```

Comment (oterral) [reply .github/workflows/nodejs.yml]: @lb1c   just a minor change before merge.

**Comment by mhaertwig** [comment]
Assigned to @oterral and @friedjoff

**Comment by oterral** [comment]
See https://github.com/geops/trafimage-maps/pull/288

**Pull request closed without merging**